tests/translate/pythonir.py

```diff
@@ -0,0 +1,57 @@
+# TODO: I think I'll need this BaseNode class but not sure yet
```

Comment:
YAGNI.

Reply:
`pythonir` seems like an odd module name to me. `python` as opposed to...?

Reply:
it's meant to hold IR that represents the python test.  But I've renamed it to `ir.py`.